concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-1034	1034	helfen	to help	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1897	HELP
+Gabelentz-1892-1074-1035	1035	Rösten	to roast	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1962	ROAST (SOMETHING)
+Gabelentz-1892-1074-1036	1036	Rudern	to row	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1849	ROW
+Gabelentz-1892-1074-1037	1037	Säen	to sow	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	748	SOW SEEDS
+Gabelentz-1892-1074-1038	1038	Sammeln	to collect	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	7	GATHER
+Gabelentz-1892-1074-1039	1039	Schälen	to peel	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	275	PEEL
+Gabelentz-1892-1074-1040	1040	Schiessen	to shoot	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1172	SHOOT
+Gabelentz-1892-1074-1041	1041	Schlagen	to beat, to strike	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2133	STRIKE OR BEAT
+Gabelentz-1892-1074-1042	1042	Schleifen, wetzen	to grind, to whet	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1043	1043	Schneiden	to cut	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1432	CUT
+Gabelentz-1892-1074-1044	1044	Spielen	to play	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1413	PLAY
+Gabelentz-1892-1074-1045	1045	Stechen	to stab, to sting	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1434	STAB
+Gabelentz-1892-1074-1046	1046	durchbohren	to pierce	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	398	PIERCE
+Gabelentz-1892-1074-1047	1047	stossen	to push	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1452	PUSH
+Gabelentz-1892-1074-1048	1048	Stehlen	to steal	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	713	STEAL
+Gabelentz-1892-1074-1049	1049	Strafen	to punish	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3649	PUNISH
+Gabelentz-1892-1074-1050	1050	Tätowiren	to tattoo	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1051	1051	Tauschen	to swap, to exchange	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2509	EXCHANGE
+Gabelentz-1892-1074-1052	1052	Theilen, vertheilen	to share	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	884	SHARE
+Gabelentz-1892-1074-1053	1053	Tragen	to carry	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	700	CARRY
+Gabelentz-1892-1074-1054	1054	Treffen	to hit a target	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3307	HIT (A TARGET)
+Gabelentz-1892-1074-1055	1055	Treiben, vertreiben	to drive, to disperse	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3290	IMPEL
+Gabelentz-1892-1074-1056	1056	Uebergeben, auslieferm	to hand over	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1057	1057	Uebertreten	to pass over, to trespass	VIII. Verba	F. Andere Verba (alphabetisch geordnet)		
+Gabelentz-1892-1074-1058	1058	Verbergen	to hide	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2486	HIDE
```

Comment:
HIDE (CONCEAL), otherwise "sich" is missing.

Reply:
See comment above. Since this is ambiguous, I think HIDE is suitable in covering both meanings.

Reply:
We have two lists with _verbergen_ mapped to HIDE (CONCEAL) so I would agree with @Schweikhard here.